xarray/tests/test_variable.py

```diff
@@ -1871,9 +1872,16 @@ def test_quantile_interpolation_deprecation(self, method) -> None:
     def test_quantile_chunked_dim_error(self):
         v = Variable(["x", "y"], self.d).chunk({"x": 2})
 
-        # this checks for ValueError in dask.array.apply_gufunc
-        with pytest.raises(ValueError, match=r"consists of multiple chunks"):
-            v.quantile(0.5, dim="x")
+        if module_available("dask", "2024.11.0"):
```

Comment:
let's add `has_dask_2024_11_0` like in https://github.com/pydata/xarray/pull/9720/files

Reply:
Sounds good, done